### DISCUSSION TITLE
UniFi OS 2.X

### DISCUSSION THREAD

**Discussion by DROEFTOETERX** [fabianishere/udm-iptv#234] [General]
Has anyone tried migrating to UniFI OS 2.X and installing udm-iptv on that? Were you able to make it work? What issues have you encountered if any?

**Comment by fabianishere** [comment]
https://github.com/fabianishere/udm-iptv/issues/212

**Comment by DROEFTOETERX** [comment]
Ah sorry, I wasn't able to find it, thanks!
